### ISSUE TITLE
Allow messages consisting only of quotes to be quoted

### ISSUE THREAD

**Issue by licaon-kter** [iNPUTmice/Conversations#4369]
What I'd like to do:
* long press message
* quote
* write some text
* send 

What I can do now:
* long press message
* copy
* long press input filed
* quote
* write some text
* send

Regarding nested quotes, I don't think this needs another level added, just kept as it is.

**Comment by Millesimus** [comment]
From the steps you are currently doing, you want to quote a quoted message (instead of copying it into the input field but without another nesting level). Imho, the easiest and most consistent way would be to bump the QUOTING_MAX_DEPTH variable to 2 (as suggested in #4148), otherwise we would have to allow nested quoting with a depth of two if and only if the message consists only of quotes.

**Comment by licaon-kter** [comment]
Bumping the MAX_DEPTH allows quoting a certain message to be quoted, as the quote depth is deeper Quote actions disappears again ( testing d4ec1eaf3878dc1f43f1bf81124731de64537bfd )

I'll reiterate:
> Regarding nested quotes, I don't think this needs another level added, just kept as it is.

`IF quote=true THEN just copy` ;)

**Comment by Millesimus** [comment]
This would make quoting behavior inconsistent, because it sometimes means "quoting" and sometimes it means "copy". This usecase would be better covered by an option like "copy-to-text-field", though this is already given with the sharing functionality (share->Conversations->current Conversation).

**Comment by licaon-kter** [comment]
I don't think that adding another | to the text really adds anything to the experience. You track the quote depth even if the content did non change?

**Comment by Millesimus** [comment]
Writing 
```
 ||| text
 ||| text
```

at least contains the information that you are referring to this message:
```
 || text
 || text
```

whereas writing
```
 || text
 || text
```
(the result of your suggestion to insert a copy and not a quote) would indicate that you are referring to this message:
```
 | text
 | text
```
If you are referring to the latter message, you should rather scroll to that message and quote it.
